plugins/wazuh-fleet/public/components/agents/details/details.tsx

```diff
@@ -8,68 +8,49 @@ import {
   EuiContextMenuPanel,
   EuiContextMenuItem,
   EuiHorizontalRule,
-  EuiTabbedContent,
+  // EuiTabbedContent,
   EuiLoadingContent,
   EuiContextMenu,
-  EuiIcon,
 } from '@elastic/eui';
-import { Agent } from '../../../../common/types';
+import { IAgentResponse } from '../../../../common/types';
+import { getAgentManagement } from '../../../plugin-services';
 import { AgentResume } from './resume';
-import { AgentDashboard } from './dashboard';
-import { AgentNetworks } from './networks';
+// import { AgentDashboard } from './dashboard';
+// import { AgentNetworks } from './networks';
 
 export interface AgentDetailsProps {
-  useDataSource: any;
-  FleetDataSource: any;
-  FleetDataSourceRepository: any;
+  indexPatterns: any;
```

Comment:
The indexPattern prop is typed as any. Using any defeats the purpose of TypeScript. Consider defining a specific type for indexPattern to improve type safety and maintainability.

import { IndexPattern } from '../../../../src/plugins/data/public';

```suggestion
  indexPattern: IndexPattern;
```

Reply:
Solved https://github.com/wazuh/wazuh-dashboard-plugins/commit/1af4ff02b310f9ddaa2420786e5062d4b8dbc416